### DISCUSSION TITLE
Enhancement: Difficulty-Specific Vocal Tracks

### DISCUSSION THREAD

**Discussion by T0paz** [FunkinCrew/Funkin#1624] [Ideas]
## What is your suggestion, and why should it be implemented?

**Current game function:** Each song has one vocal track which plays on any difficulty. The notes of the vocals correspond to the charts of the Hard difficulty for that song.

**Suggestion:** Have three different vocal tracks, each corresponding to charts of each difficulty. This should be optional for players who want this.

**Reasoning:** Depending on the player, rhythm games can be played differently in terms of how the "instructions" for what button to press are obtained. For DDR style games, players often depend on reading the charts quickly as they scroll by in one long sequence, but FNF differs by having a series of shorter patterns to copy. This style of rhythm game resembles some of the minigames from Rhythm Heaven, wherein the game communicates its "instructions" for buttons and timing through sound, but without the need to actually look at the game.

For me, I've always had trouble relying on visuals for rhythm games so DDR style games have typically been the bane of my existence. However, because FNF is set up mostly as a "copy cat" type system, I can rely on the vocal tracks alone to get the timing of the notes and quickly parse them as they scroll for the actual directions to press. The only problem is that the vocal tracks only line up with the Hard charts, so I've been playing on Hard all of the time. When I don't, the discrepancy between the voice and chart actually messes me up a lot, which makes it difficult for the wrong reasons. 

I imagine I'm not the only person who might experience this issue, and the Hard difficulty isn't for everyone. So I think adding this feature as an option would make the game more accessible to those who have a hard time looking at DDR style games. However, I do acknowledge that this would require going back and making pairs of new vocals for all existing weeks. Also, songs like Winter Horrorland have no chance of a feature like this.

**Comment by TentaRJ** [comment]
Oh no. Imagine making 2 new voice tracks for each new song.

**Comment by aflacc** [comment]
that would be cool actually. I think what you can do is just add 3 different songs which are just the same song but with different vocals and have each difficulty json for the level read a different one for the song.

**Comment by oketariq** [comment]
It would be painful to make 2 new vocal tracks for every song
